amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -99,6 +100,7 @@ export class ImagePreview extends React.Component<
               onError={this.onError}
               height="auto"
               width="100%"
+              alt=""
```

Comment:
The "sr-only" text is associated with the button, not really with the image. 
We can say that as a general rule, all buttons need to have some text inside, while the images can or cannot have an alternative text attribute.

Reply:
Ah noted that this is for the button. Is there a reason why you prefer to add an element with `sr-only` vs. using the `aria-label` button attribute? 

I'm partial to `aria-label` because it overrides whatever text gets nested in the button as well and gives us full control in every case I can currently think of, whereas with the `sr-only` approach I wonder what we would do if there was a case where it makes sense to have certain text in the button for sighted users whereas screen readers might actually need something with more detail. Thoughts?

Reply:
That's a great question!

It seems that in general the 'sr-only' class is more reliable than the aria label, here is an excerpt from (https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/): 
"the main upside to using screen reader only text over an aria-label is that the screen reader is not required to support ARIA, it’s simply reading text on the page that sighted users cannot see. While most screen readers support ARIA, you may not want to rely on it if you don’t have to."

In the case you mention, the 'sr-only' text is actually better, as sometimes the aria-label text gets completely ignored on the screen reader (check the last example on that linked website).

Reply:
I see, I actually didn't know aria support had gaps. Also found this: https://developer.paciellogroup.com/blog/2018/07/short-note-on-aria-support/

> ARIA is not tested for screen reader support before the ARIA specification is published....

> The catch is that ARIA is tested for support in accessibility APIs, but not in screen readers...

😲